### DISCUSSION TITLE
数据请求（紧急）

### DISCUSSION THREAD

**Discussion by langningchen** [XMOJ-Script-dev/XMOJ-Script#440] [General]
@boomzero 请问一下能不能给我一份现有 XMOJ-Script 用户的列表？通过邮箱发给我，谢谢！（**尽快**）

chenlangning2009@outlook.com

**Comment by boomzero** [comment]
no, wait, I forgot to send the data!

**Comment by langningchen** [comment]
现在不用了

**Comment by boomzero** [comment]
Sent
